pkg/virt-controller/services/template.go

```diff
@@ -430,6 +431,11 @@ func (t *TemplateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 		command = append(command, "--simulate-crash")
 	}
 
+	if t.clusterConfig.HyperVLayeredEnabled() {
```

Comment:
Per se, flags should protect API and not dictate behavior, TODO take the value from CR